doc/classes/LineEdit.xml

```diff
@@ -275,6 +275,9 @@
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable" default="true" keywords="readonly, disabled, enabled">
 			If [code]false[/code], existing text cannot be modified and new text cannot be added.
 		</member>
+		<member name="expand_icon" type="bool" setter="set_expand_icon" getter="is_expand_icon" default="false">
+			When enabled, the button's icon will expand/shrink to fit the LineEdit's size.
```

Comment:
You could mention what happens when this is disabled. The icon will be resized to text size.